roles/nova-common/templates/etc/nova/nova.conf

```diff
@@ -160,6 +160,11 @@ reserved_host_memory_mb = 4096
 {% endif %}
 disk_cachemodes = "network=writeback"
 
+{% if v7k.enabled %}
+iscsi_use_multipath = True
```

Comment:
- Not sure why need switch multi-path from true to false. It is kind of switching from HA configuration to configuration with SPOF(single  point of failure). So far not clear what is  cost of multi-path

- switch from true to false, may have not impacted on the volumes  attached to vms. Have to verify it.